src/client.ts

```diff
@@ -2242,6 +2232,40 @@ export class StreamChat<
     return false;
   }
 
+  _muteStatus(cid: string) {
```

Comment:
since this function only takes one cid as param, shouldn't we break out of the loop if the looped cid matches the param cid?